README.md

```diff
@@ -296,6 +296,13 @@ There are several walkthroughs to explore additional aspects of the platform:
 -   [RabbitMQ setup with Federation and Shovel plugins](https://volttron.readthedocs.io/en/latest/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.html)
 -   [Backward compatibility with the RabbitMQ message bus](https://volttron.readthedocs.io/en/latest/deploying-volttron/multi-platform/multi-platform-multi-bus.html)
 
+## Upgrade to VOLTTRON 8.x
```

Comment:
@shwethanidd and @schandrika 
Lets put this at the top of the readme hoping people will see it.  Call it Upgrading rather than upgrade.

Reply:
Updated!

Reply:
Included steps needed for bootstrap.py.